README.md

```diff
@@ -4,106 +4,66 @@ String interpolation of translated text and React components.
 
 Intended for [gettext](https://www.npmjs.com/search?q=gettext) or similar translation APIs (where the key is the intended text in the developer's locale).
 
-## Concept
+## TLDR
 
-### Tagged Template Literals
+- Two functions
+  - `gettext`
+    - Simple translation with language strng as lookupkey
+  - `ngettext`
+    - Singular/Plural aware translation with language strng as lookupkey
+- Two ways of invoking
+  - simple values
+    - The argument/string is used as the lookup id
+  - object
+    - The key is used as the lookup id
+### `gettext`
 
-This library leverages [Tagged Template Literals](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals#Tagged_template_literals) interpolation functions.
+Simple Values:
 
-Template Literals allow **substitutions**.
-
-Normally, substitutions are passed **by value**. And such a function would look like this.
-
-**(BAD)**
-```javascript
-const tag = (template, ...substitutions) => ...
-const substitution = 10;
-
-tag`Some statement that can use ${substitution}`
-// => Some statement that can use 10
-```
-
-In this form we are not able to infer a sensible **token name**.
-
-Presuming we use the substitution index we would be left with a translation like this.
-
-```
-msgid "Some statement that can use __0__"
-msgstr ...
+```js
+gettext`foo`
 ```
 
-Obviously `__0__` is **not** a good placeholder. We need something more descriptive.
-
-We need to be equally explicit about the **token name** and **token value**.
-
-**(GOOD)**
+key/val:
 
-```javascript
-tag`Some statement that can use ${{num: substitution}}`
-// => Some statement that can use 10
+```js
+gettext`Welcome to ${{location: 'hell'}}`
```

Comment:
Hmm... would like to indulge this but I would prefer not. Can we say something more positive?